imod/tests/test_protocols_imod.py

```diff
@@ -328,7 +328,8 @@ def test_fiducialAlignmentOutputCoordinates3D(self):
 
     def test_fiducialAlignmentOutputCoordinates3DSize(self):
         self.assertTrue(self.protFiducialAlignment.outputSetOfCoordinates3D.getSize() == 50 or
-                        self.protFiducialAlignment.outputSetOfCoordinates3D.getSize() == 49)
+                        self.protFiducialAlignment.outputSetOfCoordinates3D.getSize() == 49 or
+                        self.protFiducialAlignment.outputSetOfCoordinates3D.getSize() == 51)
```

Comment:
As I see, your expected value is 50, but it may be 1 unit higher or lower. I suggest to compress this assertion using a tolerance: self.assertTrue(abs(self.protFiducialAlignment.outputSetOfCoordinates3D.getSize() - 50) <= tol), where tol is equal to 1, in this case.

Reply:
Done!